python-bareos/bareos/bsock/lowlevel.py

```diff
@@ -5,12 +5,13 @@
 # Authentication code is taken from
 # https://github.com/hanxiangduo/bacula-console-python
 
-from   bareos.exceptions import *
+import bareos
```

Comment:
maybe `import bareos.exceptions` instead of `import bareos`. Otherwise the `from` imports below should probably be converted too